backend/app/services/rag/embedding/factory.py

```
@@ -0,0 +1,138 @@
+    Raises:
+        ValueError: If model not found or not an embedding model
+    """
+    # Query Model CRD from kinds table
+    # Search order: user's models -> public models (user_id=0)
+    model_kind = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "Model",
+            Kind.name == model_name,
+            Kind.namespace == model_namespace,
+            Kind.is_active == True,
```

Ruff (0.14.8)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)